### PR TITLE
Fix npm audit security vulnerabilities.

### PR DESCRIPTION
**Pull request by benwiley4000**
Hey! Thanks for this project. :)

I was unable to automatically resolve the `css` vulnerability in my own package, which is there because svg-react-loader relies on a pinned version 2.2.1 of svg-react-loader.

I upgraded all the npm audit vulnerabilities, and added a package-lock.json. With that, it might be advisable to change the pinned package.json version numbers to patch-flexible (`~`) or minor-flexible (`^`). That way it would be a bit easier to remove my own vulnerabilities in the future, without having to open a pull request. :slightly_smiling_face: I left them how they are for now.

`npm test` passes green.

I also ran `npm run karma`, although I didn't see anything happen after webpack built, so not sure what to do with that.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
   },
   "homepage": "https://github.com/jhamlet/svg-react-loader#readme",
   "dependencies": {
-    "css": "2.2.1",
+    "css": "2.2.3",
```
Comment (Fabianopb) [inline package.json]: This is already in `2.2.4`, could be worth to merge it using the latest.

File: package.json [hunk 2]
```diff
@@ -45,14 +45,14 @@
...
-    "karma": "1.7.0",
+    "karma": "3.0.0",
     "karma-chrome-launcher": "1.0.1",
     "karma-cli": "1.0.1",
     "karma-mocha": "1.3.0",
     "karma-mocha-reporter": "2.2.3",
     "karma-sourcemap-loader": "0.3.7",
-    "karma-webpack": "2.0.3",
-    "mocha": "2.5.3",
+    "karma-webpack": "3.0.1",
```
Comment (Fabianopb) [inline package.json]: Latest is `3.0.5` as of today.

**Comment by Fabianopb** [comment]
Maybe @jhamlet can give it a go after @benwiley4000 updates the PR?

**Comment by benwiley4000** [comment]
Thanks for flagging @Fabianopb. I've pushed an update. @jhamlet do you think you might have time to review soon? :slightly_smiling_face:

**Comment by jhamlet** [comment]
All.

Apologies for the delay. Been super swamped. I’ll take a look at this (and other outstanding requests) this weekend.

;-j

**Comment by jhamlet** [comment]
Merged and updated.

**Comment by benwiley4000** [comment]
@jhamlet thank you!

**Pull request merged**